docs/example.ts

```diff
@@ -82,7 +63,11 @@ const rawClaim = {
   age: 29,
 }
 
-const claim = new Kilt.Claim(ctype, rawClaim, claimer)
+const claim = new Kilt.Claim({
+  cTypeHash: ctypeHash,
+  contents: rawClaim,
+  owner: claimer.address,
+})
```

Comment:
the new constructor since 0.17.0 is 
```suggestion
const claim = Kilt.Claim.fromCTypeAndClaimContents(
  ctype,
  rawClaim,
  claimer.address,
)
```
additionally, there is a new constructor for the Ctype:
`Kilt.CType.fromCType(rawCtype)`